niworkflows/utils/misc.py

```diff
@@ -72,11 +88,26 @@ def get_template_specs(in_template, template_spec=None, default_resolution=1):
         "res", template_spec.get("resolution", default_resolution)
     )
 
+    # Verify resolution is valid
+    if fallback:
+        res = template_spec['resolution']
+        if res and not isinstance(res, list):
```

Comment:
Are you checking if `res` is falsey or specifically `None`? It could be an empty list, `0` or `''` and fail this.

```suggestion
        if res is None and not isinstance(res, list):
```

Reply:
I think your suggestion would actually break the logic.

That said, if you always listify, the intersection below (98) will still be empty with weird values. So it would be easier to just listify:

```suggestion
        if not isinstance(res, list):
```